searchlib/src/vespa/searchlib/common/bitvector.cpp

```diff
@@ -20,11 +20,11 @@ using vespalib::alloc::Alloc;
 
 namespace {
 
-void verifyContains(const search::BitVector & a, const search::BitVector & b) __attribute__((noinline));
+void verifyInclusiveStart(const search::BitVector & a, const search::BitVector & b) __attribute__((noinline));
 
-void verifyContains(const search::BitVector & a, const search::BitVector & b)
+void verifyInclusiveStart(const search::BitVector & a, const search::BitVector & b)
 {
-    if ((a.getStartIndex() < b.getStartIndex()) || (a.size() > b.size())) {
+    if (a.getStartIndex() < b.getStartIndex()) {
```

Comment:
Consider adjusting exception message to match new semantics.

Reply:
Fixed